db_connection.ts

```diff
@@ -0,0 +1,29 @@
+import { MongoClient, Db } from 'mongodb';
+
+const readEnv = (env: string) => {
+  const val = process.env[env];
```

Comment:
The validation of env variables shouldn't happen here. When you open a project and try to look for the configuration you wouldn't like it to be next to the use.
Also, a bad side effect can be observed when testing. What if you are testing a file that doesn't need db functions ? The validation wouldn't occur although you are expecting it.

Reply:
Also change variable name from var. `envVar`  > `var`
Also check [this](https://github.com/yonathan06/fastify-typescript-starter/blob/main/src/plugins/config.ts) out